packages/devtools_app/lib/src/shared/side_panel.dart

```diff
@@ -178,7 +219,25 @@ class ReleaseNotes extends AnimatedWidget {
   }
 }
 
-class ReleaseNotesController {
+class SidePanelController {
+  ValueListenable<String?> get markdown => _markdown;
+
+  final _markdown = ValueNotifier<String?>(null);
+
+  ValueListenable<bool> get isVisible => _isVisible;
+
+  final _isVisible = ValueNotifier<bool>(false);
+
+  void toggleVisibility(bool visible) {
+    _isVisible.value = visible;
+  }
+
+  set markdownText(String markdownText) {
+    _markdown.value = markdownText;
+  }
+}
+
+class ReleaseNotesController extends SidePanelController {
```

Comment:
can you actually move this class back to its own file in framework/release_notes/release_notes.dart (where it used to be)? Since you have to make a commit to kick the bots anyway :)

Reply:
You should be able to push a commit now. I just verified on one of my PRs that the tests ran as expected on the bots: https://github.com/flutter/devtools/pull/5770.

My PR will likely conflict with yours, so I will wait for yours to land and then merge